components/scream/src/physics/p3/p3_functions.hpp

```diff
@@ -32,6 +32,13 @@ struct Functions
       tabsize     = 12, // number of quantities used from lookup table
       rcollsize   = 30,
       coltabsize  = 2,  // number of ice-rain collection  quantities used from lookup table
+
+      // switch for warm-rain parameterization
+      // 1 => Seifert and Beheng 2001
+      // 2 => Beheng 1994
+      // 3 => Khairoutdinov and Kogan 2000
+      iparam      = 3,
```

Comment:
Is there any way we can turn switch variables into enums? I think they help new users understand a tad better what the var is when they later find a switch/if on that.

Reply:
Yes, we can do that. I will hold off on this for now since @pressel mentioned this selection may be going away soon.

Reply:
> @AaronDonahue and I discussed this. From that discussion, I believe the plan is remove these selections in the code, and keep only the options that are currently being used in the model. Maybe @AaronDonahue and @PeterCaldwell could confirm that this is the plan?

Yes, I agree that we should delete the versions of processes that we're not actively using as part of our port to C++. That said, @hassanbeydoun may have an opinion about switching the process implementations we're currently using.